README.md

```diff
@@ -135,7 +135,7 @@ Make sure that the PR includes the changes made by running this command.
 yarn build
 # To build TypeScript type definition files
 yarn ts:build
-# To only typecheck all TypeScript files
+# To only typecheck all TypeScript files (must have run ts:build at least once)
```

Comment:
Is this still true after all the refactoring you did? Also is package.json still up to date?

Reply:
```suggestion
# To only typecheck all TypeScript files
```